makefile.shade

```diff
@@ -699,4 +661,64 @@ functions
 
             return true;
         }
+        
+        static IEnumerable<string> GetRepositoriesToBuild() 
+        {
+            IEnumerable<string> reposToBuild = new HashSet<string>(StringComparer.OrdinalIgnoreCase) {
```

Comment:
var doesn't work well here. I am assigning an IEnumerable later but the first time the type is resolved as HashSet. I would have to cast it anyway...